reto.exs

```diff
@@ -0,0 +1,23 @@
+defmodule Recursion do
+  def print_multiple_times(msg, cantidad, cont, limit_inf, limit_sup) when cont <= limit_sup do
+    msg = msg <> "#{cont}"
+    numero = String.to_integer(msg)
```

Comment:
Remember all variables/method names and etc should be in English 
```suggestion
    number= String.to_integer(msg)
```

Reply:
This also applies for "cantidad", "limit_sup" (which could be sup_limit), "Respuesta" and the name of the file